lib/still/compiler/template_helpers.ex

```diff
@@ -44,6 +46,10 @@ defmodule Still.Compiler.TemplateHelpers do
   end
 
   def include(env, file, metadata, opts) do
+    if not input_file_exists?(file) do
+      raise "File #{file} does not exist in #{get_input_path()}"
+    end
```

Comment:
how about `ensure_file_exists!`, and move the raise into it? seems a bit cleaner to me
just a suggestion though

Reply:
👍